### ISSUE TITLE
When cwd has spaces in it, elm-test can't find elmTestOutput.js

### ISSUE THREAD

**Issue by avh4** [rtfeldman/node-test-runner#147]
If the current working directory has a space in it, like `/path/with spaces/`, then `elm-test` compiles the tests to `/path/with\ spaces/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js` (it adds a backslash in the path!) and then dies because it can't find  `/path/with spaces/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js` (with no added backslash)

```
$ elm-test
Success! Compiled 0 modules.
Successfully generated /dev/null
Success! Compiled 1 module.
Successfully generated /path\ with\ spaces/elm-testable/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js
The test run failed because it could not read the compiled Elm code:

 { Error: ENOENT: no such file or directory, open '/path with spaces/elm-testable/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/path with spaces/elm-testable/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js' }

$ ls '/path with spaces/elm-testable/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js'
ls: /path with spaces/elm-testable/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js: No such file or directory

$ ls '/path\ with\ spaces/elm-testable/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js'
/path\ with\ spaces/elm-testable/elm-stuff/generated-code/elm-community/elm-test/elmTestOutput.js
```

**Comment by avh4** [comment]
It looks like the problem might be in `node-elm-compiler`

**Comment by avh4** [comment]
Yes, the problem is https://github.com/rtfeldman/node-elm-compiler/blob/master/index.js#L176

(command line args don't need to be escaped when passed to a spawn/exec function that takes the args as an array of string.)

**Comment by rtfeldman** [comment]
@avh4 Published fixed version as 0.18.4
